### PR TITLE
Feature/e2e regression

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/pageObjects/browser-page.ts [hunk 1]
```diff
@@ -326,16 +328,18 @@ export class BrowserPage {
      * @param TTL The Time to live value of the key
      * @param value The key value
      */
-    async addJsonKey(keyName: string, TTL = ' ', value = ' '): Promise<void> {
+    async addJsonKey(keyName: string, value = ' ', TTL?: string): Promise<void> {
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/browser-page.ts:331]: Please update the description as well

File: tests/e2e/tests/regression/browser/add-keys.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        await cliPage.sendCommandInCli(commandString);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+test('Verify that user can create different types(string, number, null, array, boolean) of JSON', async t => {
+    for (let i = 0; i < jsonKeys.length; i++) {
+        await browserPage.addJsonKey(jsonKeys[i][0], jsonKeys[i][1]);
+        await t.click(browserPage.toastCloseButton);
+        await t.click(browserPage.refreshKeysButton);
+        await t.expect(await browserPage.isKeyIsDisplayedInTheList(jsonKeys[i][0])).ok('New keys is displayed');
+        if (i === jsonKeys.length - 1) {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/browser/add-keys.e2e.ts:33]: I would add comment with explanation here

File: tests/e2e/tests/regression/browser/ttl-format.e2e.ts [hunk 3]
```diff
@@ -1,54 +1,68 @@
...
+        // Create new key with TTL
+        await browserPage.addStringKey(keyName, 'test', '3');
         await t.click(browserPage.refreshKeysButton);
-        //Check that Keys has correct TTL value in keys table
-        for (let i = 0; i < keysData.length; i++) {
-            const ttlValueElement = Selector(`[data-testid="ttl-${keysData[i].keyName}"]`);
-            await t.expect(ttlValueElement.textContent).contains(ttlValues[i], `TTL value in keys table is ${ttlValues[i]}`);
-        }
+        await t.expect(await browserPage.isKeyIsDisplayedInTheList(keyName)).ok('Added key');
+        await t.wait(3000);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/browser/ttl-format.e2e.ts:65]: Reason?

**Review by ofersteinberg — CHANGES_REQUESTED** [review]

File: tests/e2e/pageObjects/browser-page.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts:331]: Chanched

File: tests/e2e/tests/regression/browser/add-keys.e2e.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/add-keys.e2e.ts]: I have re-worked this line a little bit and add comment

File: tests/e2e/tests/regression/browser/ttl-format.e2e.ts (hunk 3 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts:66]: Added a comment why wait is needed - I am checking that after TTL finishing , key is not displayed in the application

File: tests/e2e/pageObjects/browser-page.ts [hunk 4]
```diff
@@ -323,19 +325,21 @@ export class BrowserPage {
...
         await t.click(this.plusAddKeyButton);
         await t.click(this.keyTypeDropDown);
         await t.click(this.jsonOption);
         await t.click(this.addKeyNameInput);
         await t.typeText(this.addKeyNameInput, keyName);
-        await t.click(this.keyTTLInput);
-        await t.typeText(this.keyTTLInput, TTL);
         await t.click(this.jsonKeyValueInput);
         await t.typeText(this.jsonKeyValueInput, value);
+        if (TTL) {
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/browser-page.ts:339]: Another option: `if (TTL ==! undefined)`

File: tests/e2e/tests/regression/browser/ttl-format.e2e.ts (hunk 3 above)
Comment (ofersteinberg) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts:66]: So please do instead `await t.expect(browserPage.refreshKeysButton).visible('visibility of ttl' , { timeout: 5000 }` - something on that spirit, it's better approach. Waits within tests are highly not recommended.

Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts:65]: We don't auto-refresh list of keys. For this reason we need to wait for 3 seconds, than click on refresh button and only after it check key lack in the app. I have tried the way you spicified, but in doesn't work such way. If you have any concerns please let me know

Comment (ofersteinberg) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts:65]: What about implement a function of for loop with x tries?

Comment (ofersteinberg) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts:65]: As we discussed, let's add explanation here

Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts]: As we discussed, I have added a loop with refreshing and validating reduced TTL value in the test

File: tests/e2e/tests/regression/browser/ttl-format.e2e.ts [hunk 5]
```diff
@@ -58,11 +58,21 @@ test
         await deleteDatabase(ossStandaloneConfig.databaseName);
     })('Verify that Key is deleted if TTL finishes', async t => {
         // Create new key with TTL
-        await browserPage.addStringKey(keyName, 'test', '3');
+        const TTL = 5;
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/browser/ttl-format.e2e.ts:61]: Let's put higher value, such as 15

**Review by ofersteinberg — COMMENTED** [review]
Looks much better!

File: tests/e2e/tests/regression/browser/ttl-format.e2e.ts (hunk 5 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/ttl-format.e2e.ts]: Fixed

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**